src/components/Friend/RenderUserLists.js

```diff
@@ -1,29 +1,31 @@
-import React from "react";
-import { Alert, StyleSheet, View } from "react-native";
+import React, { useState } from "react";
 import { ListItem } from "react-native-elements";
 import {
   acceptFriendRequest,
   cancelFriendRequest,
   declineFriendRequest,
   removeFriend
 } from "../../services/Friend/HandleFriend";
+import {
+  blockUser,
+  unblockUser
+} from "../../services/Friend/HandleBlockedUser";
 import { contactType } from "../../constants/Contact";
 import { friendshipType } from "../../constants/Friend";
 import ContactBar from "./ContactBar";
 import ContactButtonGroup from "./ContactButtonGroup";
+import Caution from "../Miscellaneous/Caution";
+
+export default RenderUserLists = ({ type, item, navigation }) => {
+  const [isExpanded, setIsExpanded] = useState(false);
 
-export default RenderUserLists = ({
-  type,
-  items,
-  navigation,
-  expandStatus,
-  changeExpand
-}) => {
-  const buttonDetails = [
+  let buttonDetails = [
```

Comment:
should be const

Reply:
Can't use `let` because `buttonDetails` variable will be reassigned when `type === friendshipType.BLOCKED`.